_posts/2021-11-08-todo.markdown

```diff
@@ -0,0 +1,290 @@
+---
+layout: post
+title: "TODO come up with a title"
+date: 2021-11-08 03:44:05 +0100
+comments: true
+published: true
+categories: ["post"]
+tags: ["General", ".NET", "Travel"", "Startup"]
+author: Maarten Balliauw
+---
+
+For close to two years now, I've had [SpeakerTravel](https://speaker.travel/) online. It's a tool that helps conference organizers to book flights for speakers. You invite speakers, they pick their flight of choice (within a budget the organizer can specify), and the organizer can then approve and book the flight with a single click.
```

Comment:
This might read a bit better

```suggestion
For close to two years now, I've had [SpeakerTravel](https://speaker.travel/) up & running. It's a tool that helps conference organizers to book flights for speakers. You invite speakers, they pick their flight of choice (within a budget the organizer can specify), and the organizer can then approve and book the flight with a single click.
```